site/content/docs/5.0/examples/headers/index.html

```diff
@@ -0,0 +1,290 @@
+---
```

Comment:
need to custom this sample with Orange design

Reply:
will be adressed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/380